distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java

```diff
@@ -373,13 +373,15 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
     }
 
     protected Option[] configureDistribution() {
+        String unpackDirectory = System.getProperty("karaf.unpack.directory");
+        unpackDirectory = (unpackDirectory == null) ? "target/exam" : unpackDirectory;
```

Comment:
You know what. Let's wait and see if the fix I have for the Bamboo build works and if it does, we probably should just remove this whole thing.